### PR TITLE
Fix error summary

### PR DESCRIPTION
**Pull request by lucymonie**
## What are you doing in this PR?
This PR is to fix an issue with the error summary in the checkout. The problem was that the error summary was showing up with empty bullet points when there were no errors, rather than just not showing at all in this case. In addition, I noticed that the stripe card errors weren't appearing in the summary, so I have made sure they are getting passed in correctly.

[**Trello Card**](https://trello.com/c/AJ4rw7GK/3525-fix-issue-in-checkout-error-summary-where-bullet-points-still-show-after-errors-have-been-removed)

## Why are you doing this?
This arose from changes to the checkouts when implementing the design system in this PR: https://github.com/guardian/support-frontend/pull/2786

### CODE REVIEW

File: support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx [hunk 1]
```diff
@@ -269,6 +269,7 @@ const StripeForm = (props: StripeFormPropTypes) => {
     });
 
   const requestSCAPaymentMethod = (event) => {
+    console.log('called request method');
```
Comment (i-hardy) [inline support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx:272]: Stray console.log!

**Review by i-hardy — APPROVED** [review]
That stray log needs removing, otherwise this looks good!

File: support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx (hunk 1 above)
Comment (lucymonie) [reply support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx:272]: Whoops! Thanks 🦑

**Comment by lucymonie** [comment]
Thanks @i-hardy! 🎉

**Pull request merged**